lib/sqlalchemy/orm/relationships.py

```diff
@@ -176,6 +176,11 @@
     Callable[[], Iterable[_ColumnExpressionArgument[Any]]],
     Iterable[Union[str, _ColumnExpressionArgument[Any]]],
 ]
+_RelationshipBackPopulatesArgument = Union[
+    str,
+    PropComparator[Any],
+    Callable[[], "_RelationshipBackPopulatesArgument"],
```

Comment:
**Michael Bayer** (zzzeek) wrote:

it can be a string now (but correct, not a callable).     when i do "lambda" for something I make the thing it can return, and the argument that is accepted without the lambda, the same.  There's no point in having non-lambda be types X, Y and lambda return Z, Q.   both should return Q, X, Y, Z.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4831